### ISSUE TITLE
Person in multiple families

### ISSUE THREAD

**Issue by tim-pearce** [tbar0970/jethro-pmm#700]
I have children in play group where sometimes a parent brings them and sometimes a grandparent does. They have separate address and contact details.
This situation could also occur with split families.
Would it be feasible for a person to belong to multiple families or maybe for families to have multiple addresses?

**Comment by tbar0970** [comment]
Thanks for posting.  I understand the need.
Would need to think carefully about the best way to capture this.
I think having a person "in" multiple families will bring too much complexity to be worthwhile.  
The logic for determining someone's "primary family" could be "where do they sleep the most nights" or "who brings them to church".

But there is scope to capture relationships between people in other ways.  I've wondered for a while about having a new custom field type "relationship" for which the value is a reference to another person.  This could be used for various things - "other parent", "other carer", "mentor" etc.

Thoughts?

**Comment by tim-pearce** [comment]
I have a custom field Family_status which describes relationships in families I have :-
Parent, Child, Friend, Guardian, Grandparent, Stepparent, Sibling and Other

I thought I had covered all options but I had to use Other recently for Auntie

**Comment by tbar0970** [comment]
Yep.  The new field type I had in mind would actually contain a reference to a person in a different family.

Eg Jimmy (child) is in the same family as Jane (adult) because he lives with her during the week
But Jimmy's "other parent" field links to John, who is in a different family, with a different address. Jimmy lives with John on the weekends.
Might be better as a separate "relationship" concept rather than a custom field type.
Can't promise any ETA on this.
